queries/fortran/highlights.scm

```diff
@@ -31,10 +31,10 @@
   "private"
 ] @include
 
-[
-"implicit"
-(none)
-] @annotation
+; [
+; "implicit"
+; (none)
+; ] @annotation
```

Comment:
We don't have this capture group, not sure what should be replaced with.

Reply:
We have it. It's now mostly `@attribute` but we wanted to change it back to `@annotation` to have `@attribute` for html attributes to be more similar in line with Atom queries. Apparently this PR got merged before the other one. It would be attribute without the other PR.